phpseclib/Crypt/EC/BaseCurves/Binary.php

```diff
@@ -84,7 +86,7 @@ class Binary extends Base
     /**
      * Sets the modulo
      */
-    public function setModulo(...$modulo)
+    public function setModulo(...$modulo): void
```

Comment:
Should probably do `int ...$modulo` for this one

Reply:
Done